generated-docs/out/responses/body.md

```diff
@@ -86,6 +86,24 @@ basicRequest
 
 A number of JSON libraries are supported out-of-the-box, see [json support](../json.md).
 
+Sometimes it might be useful to model some http error responses right away. We can do that using `asTypedErrors` method:
+```scala
+case class SuccessModel(...)
+sealed trait MyErrorModel
+case class Conflict(...) extends MyModel
```

Comment:
Typo. Seems should be `case class Conflict(...) extends MyErrorModel`

Reply:
Thanks, fixed